### PR TITLE
Added initial config with sentry to report errors

### LINKED ISSUE #226: Add integration with Sentry

### PR DESCRIPTION
**Pull request by monkin77**
closes #226

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -10,4 +10,7 @@ REACT_APP_API_HOSTNAME="https://localhost:8087"
 ANALYTICS_ID=
 
 # GeoLocation Rapid Api key
-GEO_API_KEY="APIKEY"
\ No newline at end of file
+GEO_API_KEY="APIKEY"
+
+# Sentry dsn
+SENTRY_DSN=
```
Comment (imnotteixeira) [inline .env:16]: Add newline at the end of file

**Review by imnotteixeira — COMMENTED** [review]
What is the added value of Sentry, since we are already posting metrics to google analytics, including the errors?

**Comment by monkin77** [comment]
> What is the added value of Sentry, since we are already posting metrics to google analytics, including the errors?

The main reason was to be notified when the server went down

**Comment by imnotteixeira** [comment]
I'd go with this instead. https://niaefeup.slack.com/apps/A9Z9ZNFE1-uptimebot?tab=more_info 

Adding sentry seems a bit overkill, and it adds up to the dependencies/bundle size, which makes the pages load slower

**Comment by monkin77** [comment]
> I'd go with this instead. https://niaefeup.slack.com/apps/A9Z9ZNFE1-uptimebot?tab=more_info
> 
> Adding sentry seems a bit overkill, and it adds up to the dependencies/bundle size, which makes the pages load slower

sounds good

**Pull request closed without merging**